src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```
@@ -332,60 +308,15 @@ Unix support is disabled by default. See https://aka.ms/systemdrawingnonwindows
-    <Compile Include="System\Drawing\Printing\PreviewPrintController.Unix.cs" />
-    <Compile Include="System\Drawing\Printing\PrintController.Unix.cs" />
-    <Compile Include="System\Drawing\Printing\PrintDocument.Unix.cs" />
-    <Compile Include="System\Drawing\Printing\PrinterSettings.Unix.cs" />
-    <Compile Include="System\Drawing\Printing\PrintEventArgs.Unix.cs" />
-    <Compile Include="System\Drawing\Printing\PrintPageEventArgs.Unix.cs" />
-    <Compile Include="System\Drawing\Printing\StandardPrintController.Unix.cs" />
-    <Compile Include="System\Drawing\Bitmap.Unix.cs" />
-    <Compile Include="System\Drawing\BufferedGraphics.Unix.cs" />
-    <Compile Include="System\Drawing\BufferedGraphicsManager.Unix.cs" />
-    <Compile Include="System\Drawing\BufferedGraphicsContext.Unix.cs" />
-    <Compile Include="System\Drawing\macFunctions.cs" />
-    <Compile Include="System\Drawing\Font.Unix.cs" />
-    <Compile Include="System\Drawing\FontFamily.Unix.cs" />
-    <Compile Include="System\Drawing\GdiplusNative.Unix.cs" />
-    <Compile Include="System\Drawing\GdiPlusStreamHelper.Unix.cs" />
-    <Compile Include="System\Drawing\LibX11Functions.cs" />
-    <Compile Include="System\Drawing\LocalAppContextSwitches.Unix.cs" />
-    <Compile Include="System\Drawing\MarshallingHelpers.cs" />
-    <Compile Include="System\Drawing\Image.Unix.cs" />
-    <Compile Include="System\Drawing\Pen.Unix.cs" />
-    <Compile Include="System\Drawing\Region.Unix.cs" />
-    <Compile Include="System\Drawing\SRDescriptionAttribute.cs" />
-    <Compile Include="System\Drawing\SystemIcons.Unix.cs" />
-    <Compile Include="System\Drawing\ToolboxBitmapAttribute.Unix.cs" />
-    <Compile Include="System\Drawing\Drawing2D\AdjustableArrowCap.Unix.cs" />
-    <Compile Include="System\Drawing\Drawing2D\CustomLineCap.Unix.cs" />
-    <Compile Include="System\Drawing\Drawing2D\GraphicsPath.Unix.cs" />
-    <Compile Include="System\Drawing\Text\PrivateFontCollection.Unix.cs" />
-    <Compile Include="System\Drawing\LibraryResolver.cs" />
-    <Compile Include="Interop\Unix\Interop.Libraries.cs" />
-    <EmbeddedResource Include="Resources\System\Drawing\Error.ico">
-      <LogicalName>placeholder.ico</LogicalName>
+    <EmbeddedResource Include="Resources\System\Drawing\DefaultComponent.bmp">
+      <LogicalName>System.Drawing.DefaultComponent.bmp</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Resources\System\Drawing\Printing\PrintDocument.bmp" />
+    <EmbeddedResource Include="Resources\System\Drawing\ShieldIcon.ico">
+      <LogicalName>System.Drawing.ShieldIcon.ico</LogicalName>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0')) and '$(TargetPlatformIdentifier)' != ''">
+  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0')) and '$(TargetsAnyOS)' != 'true'">
```

The `TargetsAnyOS` property doesn't exist anymore and the above version is less verbose and does the same job.

```suggestion
  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0-windows'))">
```